Tests/FunctionalTests/Sources/EarlGrey.swift

```diff
@@ -39,7 +39,7 @@ public func GREYAssertTrue(@autoclosure expression: () -> BooleanType, reason: S
 public func GREYAssertFalse(@autoclosure expression: () -> BooleanType, reason: String) {
   GREYAssert(!expression().boolValue,
     reason,
-    details: "Expected the boolean expression to be true")
```

Comment:
This file is present is in multiple places in the project. It would be great if you could replicate these changes in the other EarlGrey.swift files as well.